docs/articles/multi-agent-runner-helm-chart.md

```diff
@@ -44,7 +44,7 @@ Use the following command to install the Helm Chart from OCI Registry:
      --set 'runner.orgId=<your:tkcorg_:organization_id>' \
      --set 'runner.secret=<your:tkckey_run_:key>' \
      --set 'cloud.url=agent.testkube.io:443' \
-     my-runner oci://registry-1.docker.io/kubeshop/testkube-enterprise --version <version>
+     my-runner oci://registry-1.docker.io/kubeshop/testkube-runner --version <version>
```

Comment:
were these instructions correct? 🤔 testkube-enterprise is another chart or no? I was double checking the `--set` flags but these values seem fine